service/accounts/openstack_manager.py

```diff
@@ -201,8 +240,19 @@ def init_security_group(self, username, password, project_name,
                             security_group_name, rules_list):
         # 4.1. Update the account quota to hold a larger number of
         # roles than what is necessary
-        user = self.user_manager.keystone.users.find(name=username)
-        project = self.user_manager.keystone.tenants.find(name=project_name)
+        # user = user_matches[0]
+        # -- User:Keystone rev.
+        kwargs = {}
+        if self.identity_version > 2:
+            kwargs.update({'domain': 'default'})
+        user_matches = [u for u in self.user_manager.keystone.users.list(**kwargs) if u.name == username]
+        if not user_matches or len(user_matches) > 1:
+            raise Exception("User maps to *MORE* than one account on openstack default domain! Ask a programmer for help here!")
+        user = user_matches[0]
+        kwargs = {}
```

Comment:
Do we _mean_ to reset `kwargs` here? Does project expect `'domain_id'` and not `'domain'`?